eng/tools/openapi-diff-runner/src/utils/common-utils.ts

```diff
@@ -1,5 +1,6 @@
 import { FilePosition } from "../types/message.js";
 import { Context } from "../types/breaking-change.js";
+import { existsSync, readFileSync } from "node:fs";
```

Comment:
Do you need to use sync APIs, or can you use async (and convert the call chain to async)?  Sync IO should only be used in exceptional cases in JS.